src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -18,9 +18,23 @@ public static partial class Certificates
         {
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
-            private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
+
+            private static Mutex m;
             private const int MutexTimeout = 120 * 1000;
 
+            static Certificates()
+            {
+                if (PlatformDetection.IsUap)
```

Comment:
`PlatformDetection.IsWinRT` would apply better here, right? As `IsUap`  => `IsWinRT || `IsNetNative`, and AFAIK the Mutex issue is only in CoreCLR as it was already fixed for CoreRT.